lib/appbundler/cli.rb

```diff
@@ -9,10 +9,12 @@ class CLI
     banner(<<-BANNER)
 * appbundler #{VERSION} *
 
-Usage: appbundler APPLICATION_DIR BINSTUB_DIR
+Usage: appbundler BUNDLE_DIR BINSTUB_DIR [GEM_NAME] [GEM_NAME] ...
```

Comment:
@lamont-granquist nope, BUNDLE_DIR and APPLICATION_DIR mean the same thing. GEM_NAME defaults to File.basename(BUNDLE_DIR), which is equivalent to before as well.